src/styles/main.scss

```diff
@@ -1,3 +1,204 @@
+@import './utils/variables';
+@import './blocks/stars';
+
+*{
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	color: #060b35;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)